package.json

```diff
@@ -52,10 +52,10 @@
     "body-parser": "1.15.0",
     "dateformat": "1.0.12",
     "amqplib": "^0.5.1",
-    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "https://github.com/dcalvoalonso/iotagent-node-lib.git#task/exportConfigNgsi2",
     "logops": "1.0.0-alpha.7",
     "mqtt": "1.7.0",
-    "request": "^2.69.0",
+    "request": "2.39.0 - 2.81.0",
```

Comment:
Is this change really required? We should try to change as little dependencies as possible. I understand that the change in nock may be needed if the mock included in the new tests require so, but what about the change in request?

Reply:
This change is require so that unit tests are passed with modern versions of NodeJs. As you can see, the same versions are specified in https://github.com/telefonicaid/iotagent-node-lib/blob/master/package.json#L45. This change was introduced by https://github.com/telefonicaid/iotagent-node-lib/pull/560 to fix unit tests.

Reply:
Ok. Makes sense. NTC